examples/tree.py

```diff
@@ -1,36 +1,21 @@
-from dataclasses import dataclass
-from benda import bjit, u24
-
-@dataclass
-class Leaf:
-    value: u24
-
-
-@dataclass
-class Node:
-    left: 'Tree'
-    right: 'Tree'
-
-
-Tree = Node | Leaf
```

Comment:
keep the expected code that we'll have working in the future commented

Reply:
Ok

Reply:
Now the old example works properly.